File: 0145/Solution.go
/**
 * Definition for a binary tree node.
 * type TreeNode struct {
 *     Val int
 *     Left *TreeNode
 *     Right *TreeNode
 * }
 */
func postorderTraversal(root *TreeNode) []int {
    var res []int
    if root != nil {
        res = append(res, postorderTraversal(root.Left)...)
        res = append(res, postorderTraversal(root.Right)...)
        res = append(res, root.Val)
    }
    return res
}

File: 0145/Solution.js
/**
 * Definition for a binary tree node.
 * function TreeNode(val, left, right) {
 *     this.val = (val===undefined ? 0 : val)
 *     this.left = (left===undefined ? null : left)
 *     this.right = (right===undefined ? null : right)
 * }
 */
/**
 * @param {TreeNode} root
 * @return {number[]}
 */
var postorderTraversal = function(root) {
    const res = [];
    const postOrder = (root) => {
        if(root !== null){
            postOrder(root.left);
            postOrder(root.right);
            res.push(root.val);
        }
        return res;
    }

    return postOrder(root)

};
